### ISSUE TITLE
i wanna help u ;-;

### ISSUE THREAD

**Issue by ghost** [Leo310/Botnet#17]
is there any way I can help with this project? kkkkkkkkkk

**Comment by Leo310** [comment]
Yeah sure. Would be great 😃. The question is how much effort you want to put into it and how well your C++ understanding is.
This project is still in the beginning of development so you can quickly catch up I guess. I could give you an overview if you want to contribute. 
Thanks in advance 👍

**Comment by ghost** [comment]
bro, my commitment to the project is as much as I can and my knowledge in c ++ is good po kkkkkkkkkk trust

**Comment by ghost** [comment]
the real problem is that I didn't understand his organization

**Comment by ghost** [comment]
despite being new, honestly, I didn't understand his architecture, but one way or another I want to help you

brazil love u as i love u KKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

**Comment by Leo310** [comment]
> bro, my commitment to the project is as much as I can and my knowledge in c ++ is good po kkkkkkkkkk trust

coooool 👍 


> the real problem is that I didn't understand his organization

what do you mean? what organization?




> despite being new, honestly, I didn't understand his architecture, but one way or another I want to help you
> 
> brazil love u as i love u KKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

Do you mean the architecture of this project? Sickkk. Do you have discord?
Love u too man

**Comment by ghost** [comment]
yeah, i've [ur0mul0u#1788] tell 'em

**Comment by Leo310** [comment]
yo i send you a friend request
